ajax/libs/spscript/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "spscript",
+  "version": "2.0.0",
+  "description": "SharePoint 2013 Rest Api Wrappers",
+  "filename": "dist/v2/spscript.min.js",
+   "repository": {
+    "type": "git",
+    "url": "https://github.com/DroopyTersen/spscript"
+  },
+  "keywords": [
+    "sharepoint",
+    "rest",
+    "droopy",
+    "office",
+    "office-365"
+    "office365"
+  ],
+  "author": "Andrew Petersen",
+  "bugs": {
+    "url": "https://github.com/DroopyTersen/spscript/issues"
+  },
+  "homepage": "https://github.com/DroopyTersen/spscript"
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Or you can use https://droopytersen.github.io/spscript/ as `homepage`.